### PR TITLE
feat: introducing feed filter scroll experiment

### PR DESCRIPTION
**Pull request by rebelchris**
### Describe what this PR does
- Introduces the new scroll feed filter experiment
- In the current implementation it technically means it always shows directly on the feed (due to how we assign the new load)

Version 1:
Screenshot 2022-12-16 at 13 13 09

Version 2:
Screenshot 2022-12-16 at 13 13 32

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| New | impression  | target_type: scroll block, target_id: [V1, V2] |
| New | click scroll block  | target_id: [V1, V2] |

## Manual Testing

> Note: Only shows up on desktop sizes

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-806 #done

### CODE REVIEW

File: packages/shared/src/components/ScrollFeedFiltersOnboarding.tsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    });
+  }, []);
+
+  return (
+    <div
+      className={classNames(
+        'flex fixed bottom-0 left-60 z-3 gap-16 justify-center items-center w-full pr-40',
+        versionToContainerClassName[version],
+      )}
+      style={{
```
Comment (el) [inline packages/shared/src/components/ScrollFeedFiltersOnboarding.tsx:51]: There seems to be an issue with the background color. It should not have a hard stop
image

File: packages/shared/src/components/ScrollFeedFiltersOnboarding.tsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+        background:
+          'linear-gradient(90deg, rgba(4, 19, 30, 1) 0%, rgba(4, 19, 30, 0.605797) 100%)',
+      }}
```
Comment (el) [inline packages/shared/src/components/ScrollFeedFiltersOnboarding.tsx:54]: Same issue on v2
image

File: packages/shared/src/components/ScrollFeedFiltersOnboarding.tsx [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+            versionToButtonClassName[version],
+          )}
+          buttonSize="large"
+          onClick={onInitializeOnboarding}
+        >
+          {versionToButtonText[version]}
+        </Button>
+      </div>
+      {version === ScrollOnboardingVersion.V2 && (
+        <img
```
Comment (el) [inline packages/shared/src/components/ScrollFeedFiltersOnboarding.tsx:87]: Do we really need an image here?

There seems to be an issue with the font 
image

File: packages/shared/src/components/ScrollFeedFiltersOnboarding.tsx [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+          src={cloudinary.feedFilters.scroll}
```
Comment (el) [inline packages/shared/src/components/ScrollFeedFiltersOnboarding.tsx:88]: We can hide the right image if the screen width is small
image

File: packages/shared/src/components/ScrollFeedFiltersOnboarding.tsx [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+            'text-white btn-primary-cabbage ',
+            versionToButtonClassName[version],
+          )}
+          buttonSize="large"
+          onClick={onInitializeOnboarding}
+        >
+          {versionToButtonText[version]}
+        </Button>
+      </div>
+      {version === ScrollOnboardingVersion.V2 && (
```
Comment (el) [inline packages/shared/src/components/ScrollFeedFiltersOnboarding.tsx:86]: V2 covers the whole page if the screen height is small
image

File: packages/shared/src/components/ScrollFeedFiltersOnboarding.tsx [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+            />
+          </div>
+        )}
+        {version === ScrollOnboardingVersion.V2 && (
+          <h2 className="mb-6 font-bold typo-title2">
+            Supercharge your feed with
+            <br /> content you&apos;ll love reading!
+          </h2>
+        )}
+        <Button
```
Comment (el) [inline packages/shared/src/components/ScrollFeedFiltersOnboarding.tsx:75]: I think the user should be able to dismiss as well :/

**Review by el — CHANGES_REQUESTED** [review]

File: packages/shared/src/components/ScrollFeedFiltersOnboarding.tsx (hunk 3 above)
Comment (rebelchris) [reply packages/shared/src/components/ScrollFeedFiltersOnboarding.tsx:87]: It's per design, the font issue is known awaiting a new export from design.

File: packages/shared/src/components/ScrollFeedFiltersOnboarding.tsx (hunk 4 above)
Comment (rebelchris) [reply packages/shared/src/components/ScrollFeedFiltersOnboarding.tsx:88]: Ah that might be a good one for now.

File: packages/shared/src/components/ScrollFeedFiltersOnboarding.tsx (hunk 6 above)
Comment (rebelchris) [reply packages/shared/src/components/ScrollFeedFiltersOnboarding.tsx:75]: For now that's intended for the experiment.

**Review by el — APPROVED** [review]
Great work :shipit:

**Pull request merged**